Dockerfile

```diff
@@ -1,24 +1,22 @@
-FROM alpine:3.5
+FROM nginx:1.13.0-alpine
 
 MAINTAINER Alt Three <support@alt-three.com>
 
 EXPOSE 8000
 CMD ["/sbin/entrypoint.sh"]
-
 ARG cachet_ver
 ENV cachet_ver ${cachet_ver:-master}
 
-ENV NGINX_VERSION 1.10.3-r0
+#ENV NGINX_VERSION 1.12.0-r1
```

Comment:
Can go ahead and remove these lines fully, no need to comment them out -- we have `git` for history if we need this again :)